Source/ARTRealtime.m

```diff
@@ -855,7 +859,7 @@ - (void)onConnected:(ARTProtocolMessage *)message {
     }
 
     _resuming = false;
-    [self setExpectedSuspensionTime];
+    _connectionLostAt = nil;
```

Comment:
Why do this both here and on the CONNECTED transition side effects?

Reply:
Same as https://github.com/ably/ably-cocoa/pull/917#discussion_r369057510. Thanks!